UM/Mesh/MeshBuilder.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2015 Ultimaker B.V.
+# Copyright (c) 2017 Thomas Karl Pietrowski
```

Comment:
Uhm, what?
Changing the year is a good idea, but why change the copyright holder?

Reply:
Whoops. I didn't even notice that. I brought up updating license stuff at a meeting once and we agreed to update it when we see it, but otherwise not worry about it.

This is where a contributor license agreement comes in handy...

Reply:
Aren't credits are automatically given by Git? You can see exactly who did what and when. I think it's a bit overkill to add all this extra info to the core repository, a LOT of people worked on it in the end.

Reply:
I heard other things when Ultimaker changed the license. The number of contributors was not that much as expected if I recall correctly.
Sure, but in my opinion, there are not many people who care about Git. But on the other hand, putting the credits on top of the file is also a bit too much. I have no idea how to get a good balance for this.

Reply:
Over 100 contributors according to GitHub: https://github.com/Ultimaker/Cura/graphs/contributors. And contributors contribute under the existing license (if they wouldn't agree with it, they wouldn't contribute). Of course, a new file that's created by someone won't contain the Ultimaker copyright header, but given that Ultimaker employees created most of them, it's logical that most do.

Reply:
After this got merged I couldn't remember that I ever added this line. After seeing it again, I slightly remember that I wanted to make a joke out of it. But I see how delicate this topic seems to be after adding on top my opinion here.

Therefore because I'm not interested to annoy people over here more than needed and wanted:
Please excuse my big mouth and remove this line. :pray: